sds/src/proximity_keywords.rs

```diff
@@ -226,11 +241,33 @@ impl Default for Metrics {
     }
 }
 
+fn compile_keywords_pattern(keyword_patterns: Vec<(Ast, Ast)>) -> (String, String) {
+    let content_pattern = Ast::Alternation(Alternation {
```

Comment:
I would expect the `Ast::Alternation` to be produced by the `compile_keywords_to_ast` function, since that's part of the Ast, and it doesn't look like that function is used without it anywhere else?

This function would just be calling `.to_string()`, and could probably just be inlined where it's used

Reply:
Agreed 👍

Reply:
[Inline compile_keyword_patterns fn](https://github.com/DataDog/dd-sensitive-data-scanner/pull/52/commits/94bdea5ac283933f09e3bf0f1bbcdf90141e36b4)